frontend/src/abs/lang/abslang.abs

```diff
@@ -50,6 +50,10 @@ export Inv, Pre, Post;
 export Annotation, TypeAnnotation, LocationType, Far, Near, Somewhere, Infer;
 export COG, Plain, Final, Atomic;
 export HTTPName, HTTPCallable;
+export Expansion, ExpansionCall;
```

Comment:
Do we need to export these?  Is there anything a modeler (user of the language) can do with these annotations?

Reply:
The annotations could be useful for static analysis and IDE support, but modelers should, indeed, not use them. However, not exporting the annotation types leads to type-checking errors for the generated annotations. Is there any way I don't know about to circumvent that?

Reply:
If we can keep them purely an implementation detail, you can use a similar approach to Lars, see file `ASTPreProcessor.java` line 177, and grep for `FUNCTIONSELECTOR` to see where he uses it

Reply:
I see. The problem with using this approach is that it just "marks" a function as an accessor function. The Expansion and ExpansionCall annotations also refer to each other (e.g. `[ExpansionCall: list[1]]` refers to a function with the `[Expansion: 1]` annotation). As I see it, there is no way of doing that with Lars' approach and still be able to identify the annotation types.